packages/dht/src/connection/websocket/WebsocketConnector.ts

```diff
@@ -183,33 +183,29 @@ export class WebsocketConnector {
         for (const reattempt of range(ENTRY_POINT_CONNECTION_ATTEMPTS)) {
```

Comment:
Could rename the `noServerConnectivityResponse` variable at line `174` as it is not accurate anymore. Or maybe the use case `(this.abortController.signal.aborted)`, could just throw an Error as we are not supposed to run connectivity checks when the application has been stopped (we do that only while we start the application, i.e. before it is allowed to call `stop()`).